src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -129,96 +32,113 @@ export function ActionsPanel() {
+    // clickable
     return (
-      <Button onClick={() => addNavPath(key)} key={key}>
-        <FolderIcon />
-        <p>{key}</p>
-      </Button>
+      <Box flex={1}>
+        {paths.map((path, i) => {
+          const isLast = i === paths.length - 1;
+          return (
+            <Text
+              key={path}
+              onClick={isTopLevel ? undefined : () => goToPath(path)}
+              display={'inline'}
+              style={{ cursor: isTopLevel ? 'default' : 'pointer' }}
+            >
+              {path}
+              {!isLast ? '/' : ''}
+            </Text>
+          );
+        })}
+      </Box>
     );
   }
 
-  function getFoldersContent(level: FolderContent) {
-    return Object.keys(level.folders)
-      .sort()
-      .map((key) => folderButton(key));
-  }
-
-  function getActionContent(level: FolderContent) {
-    if (level.actions.length === 0) {
-      return null;
-    }
-
-    return level.actions.map((action: Action) => (
-      <ActionsButton key={`${action.identifier}Action`} action={action} />
-    ));
-  }
-
-  function getBackButton() {
-    if (navigationPath !== '/') {
-      return (
-        <Button onClick={goBack} key={'backbtn'}>
-          Back
-        </Button>
-      );
-    }
-    return null;
-  }
-
   if (actionLevel === undefined) {
     return <p>Loading...</p>;
   }
-  // TODO (ylvse 2024-10-10) Check what this check is meant to do?
-  // In the js it is actionlevel.length === 0. But how can it have a length?
-  const isEmpty = actionLevel === null;
-  const actions = isEmpty ? <div>No Actions</div> : getActionContent(actionLevel);
-  const folders = isEmpty ? <div>No Folders</div> : getFoldersContent(actionLevel);
-  const backButton = getBackButton();
+
   return (
     <ScrollArea h={'100%'}>
-      <Flex>
-        {backButton}
-        <p>{`${displayedNavigationPath}`}</p>
-      </Flex>
-      <FilterList>
-        <FilterList.InputField placeHolderSearchText={'Search for an action...'} />
-        <FilterList.Favorites>
-          <Grid>
-            {folders}
-            {actions}
-          </Grid>
-        </FilterList.Favorites>
-        <FilterList.Data<Action>
-          data={displayedActions}
-          renderElement={(action: Action) => (
-            <ActionsButton key={`${action.identifier}Filtered`} action={action} />
+      <Container>
+        <Group gap={'xs'} my={'xs'}>
+          {navigationPath !== '/' && (
+            <ActionIcon onClick={goBack} key={'backbtn'}>
+              <BackArrowIcon />
+            </ActionIcon>
           )}
-          matcherFunc={generateMatcherFunctionByKeys(['identifier', 'name', 'guiPath'])}
-        />
-      </FilterList>
+          {displayNavigationPath()}
+        </Group>
+        <FilterList>
+          <FilterList.InputField placeHolderSearchText={'Search for an action...'} />
+          <FilterList.Favorites>
+            <DynamicGrid gutter={'xs'} minChildSize={150} className={'actionsPanelGrid'}>
```

Comment:
```suggestion
            <DynamicGrid gutter={'xs'} minChildSize={170} className={'actionsPanelGrid'}>
```
I would suggest increasing this min size slightly, to give the text for the folders a bit more space

Reply:
Also, there should not be a fixed classname here. Instead we should use CSS Modules
https://mantine.dev/styles/css-modules/